src/autopas/containers/cellPairTraversals/VerletListsTraversal.h

```diff
@@ -0,0 +1,54 @@
+/**
+ * @file VerletListsTraversal.h
+ * @author nguyen
+ * @date 16.09.18
+ */
+
+#pragma once
+
+#include <array>
+#include <utility>
+#include <vector>
+
+namespace autopas {
+
+/**
+ * A verlet list traversal.
+ * This class handles traversals through the cell structures with neighbor lists.
+ * Derived classes handle the order through which the cells are traversed.
+ */
+template <class PairwiseFunctor, bool useNewton3>
+class VerletListsTraversal {
+ public:
+  /**
+   * Constructor of the verlet traversal.
+   * @param pairwiseFunctor The functor that defines the interaction of two particles.
+   */
+  VerletListsTraversal(PairwiseFunctor *pairwiseFunctor) : _pairwiseFunctor(pairwiseFunctor) {}
+
+ protected:
+  /**
+   * iterate over the verlet list of a given cell
+   * @param verlet
+   * @param cellIndex
+   */
+  template <class Particle>
+  inline void iterateVerletListsCell(std::vector<std::vector<std::pair<Particle *, std::vector<Particle *>>>> &verlet,
```

Comment:
this function should actually be some sort of cell functor for verlet lists. @FG-TUM: correct?

Reply:
needed to later on be able to differentiate between aos and soa

Reply:
#145